tests/nfs/nfs_operations.py

```diff
@@ -88,6 +104,16 @@ def cleanup_cluster(clients, nfs_mount, nfs_name, nfs_export):
     if not isinstance(clients, list):
         clients = [clients]
 
+    # Check nfs coredump
+    if ceph_cluster_obj:
+        nfs_nodes = ceph_cluster_obj.get_nodes("nfs")
+        coredump_path = "/var/lib/systemd/coredump"
+        for nfs_node in nfs_nodes:
+            if not check_coredump_generated(nfs_node, coredump_path, setup_start_time):
+                raise NfsCleanupFailed(
+                    f"Coredump grenerated for node {nfs_node.hostname} after steup"
```

Comment:
rephrase to 
"Coredump generated post execution of the current test case"

Reply:
Thanks @pranavprakash20 
Updated